modular_skyrat/modules/implants/code/augments_arms.dm

```diff
@@ -219,6 +219,81 @@
 	obj_flags |= EMAGGED
 	return TRUE
 
+// Razorwire implant, long reach whip made of extremely thin wire, ouch!
+
+/obj/item/melee/razorwire
+	name = "implanted razorwire"
+	desc = "A long length of cutting wire so impossibly thin that it causes grevious wounds in anything you slash with it. \
```

Comment:
desc = "A long length of monomolecular filament, built into the back of your hand. Impossibly thin and flawlessly sharp, it should slice through organic materials with no trouble; even from a few steps away. However, results against anything more durable will heavily vary."